### ISSUE TITLE
Not seeing tracebacks in device logs when Rollbar is enabled

### ISSUE THREAD

**Issue by mgalgs** [rollbar/rollbar-flutter#23]
I've been using the recently released `0.2.0-beta` build and am successfully getting traces in my Rollbar account! :tada: :tada: 

However, when running my app under Rollbar I'm not seeing the traces in my device logs in my IDE. This is happening both on Android (under Android Studio) as well as iOS (under XCode).

If I run my app without Rollbar I do see the tracebacks in my local logs.

I've configured Rollbar as follows:

```dart
Future<void> main() async {
  var config = (ConfigBuilder('my client id')
    ..environment = 'development'
    ..codeVersion = '0.1.0'
    ..handleUncaughtErrors = true
    ..includePlatformLogs = true)
      .build();

  await RollbarFlutter.run(config, (rollbar) {
    runApp(MyApp());
  });
}
```

If I set `handleUncaughtErrors` to `false` then I do see the errors in my local logs, but not in Rollbar... Is there a way to log uncaught errors to Rollbar and then re-raise them so that I get them in both locations?

**Comment by akornich** [comment]
@mgalgs , thanks! I assume the same behavior was with v0.1.0-beta as well. Correct?

**Comment by mgalgs** [comment]
@akornich  v0.1.0-beta didn't have null safety support so I was never able to test it.

**Comment by mgalgs** [comment]
By the way, since filling this issue I've noticed a few exceptions that *are* showing up in both places... 🤔 I'll try to snag one next time to see what the different is between the example above and the ones where the stacktrace shows up in the console as well as rollbar.

**Comment by akornich** [comment]
yes, please. let's pin it down if there is an issue here.
regardless optionality of handling unhandled exceptions by this SDK: in other Rollbar SDKs we do have a config flag that allows an SDK to either report a processed unhandled exception by sending it to rollbar.com and swallowing it or reporting it to rollbar.com and rethrowing.
it is considered our SDKs standard feature and it is definitely will be coming to this SDK as well.

**Comment by mgalgs** [comment]
Just reproduced this by calling a method that takes an int but passing in a string:

```dart
void showOrderDetailsScreen(int pk) async {
}

...elsewhere...
showOrderDetailsScreen("42");
```

I do see a Rollbar item:

![image](https://user-images.githubusercontent.com/152014/165198729-eddc73db-bc62-4609-92e3-8ee71dc646a8.png)

But nothing in the console.

Wondering if it's related to the `async`?

**Comment by matux** [comment]
@mgalgs This should be fixed in the upcoming 0.3.0 release!
